Storage/src/Connection/Rest.php

```diff
@@ -113,6 +115,7 @@ public function projectId()
      */
     public function deleteAcl(array $args = [])
     {
+        $args['restRetryFunction'] = $this->getRestRetryFunction('bucket_acl', 'delete', $args);
```

Comment:
So, the idea is to abstract out this addition of `restRetryFunction` into another to avoid repetition.
```suggestion
        $this->sendRequest('bucket_acl', 'delete', $args);
```

The `sendRequest` method can add the new parameter to args and then continue the call to `$this->send`.

Reply:
Okay, will do that.

Reply:
Will do this once we find a way to tackle the hmacKeys and serviceAccount management.